packages/state-transition/src/util/epochShuffling.ts

```diff
@@ -99,6 +99,6 @@ export function computeEpochShuffling(
     shuffling,
     committees,
     committeesPerSlot,
-    dependantRoot: toHex(attesterShufflingDecisionRoot(state, epoch - 1)),
```

Comment:
@dapplion this works fine in my test, please try again at your side

Reply:
Thank you! Don't see the `warn: Potential **** epoch attester duties reorg` in either epoch 0,1,2+